library/edihistory/test_edih_835_accounting.php

```diff
@@ -217,6 +217,198 @@ function edih_835_accounting($segments, $delimiters)
+	            $out[$ck]['clp'][$i]['patient_fname'] = (isset($sar[4]) && $sar[4]) ? trim($sar[4]) : "";
+	            $out[$ck]['clp'][$i]['patient_mname'] = (isset($sar[5]) && $sar[5]) ? trim($sar[5]) : "";
+	            $out[$ck]['clp'][$i]['patient_member_id'] = (isset($sar[9]) && $sar[9]) ? trim($sar[9]) : "";
+	        } elseif ($id == 'IL') {
+				// IL = Insured or Subscriber
+	            $out[$ck]['clp'][$i]['subscriber_lname'] = (isset($sar[3]) && $sar[3]) ? trim($sar[3]) : "";
+	            $out[$ck]['clp'][$i]['subscriber_fname'] = (isset($sar[4]) && $sar[4]) ? trim($sar[4]) : "";
+	            $out[$ck]['clp'][$i]['subscriber_mname'] = (isset($sar[5]) && $sar[5]) ? trim($sar[5]) : "";
+	            $out[$ck]['clp'][$i]['subscriber_member_id'] = (isset($sar[9]) && $sar[9]) ? trim($sar[9]) : "";
+	        } elseif ($id == '82') {
+		        // 82 = Rendering Provider
+	            $out[$ck]['clp'][$i]['provider_lname'] = (isset($sar[3]) && $sar[3]) ? trim($sar[3]) : "";
+	            $out[$ck]['clp'][$i]['provider_fname'] = (isset($sar[4]) && $sar[4]) ? trim($sar[4]) : "";
+	            $out[$ck]['clp'][$i]['provider_mname'] = (isset($sar[5]) && $sar[5]) ? trim($sar[5]) : "";
+	            $out[$ck]['clp'][$i]['provider_member_id'] = (isset($sar[9]) && $sar[9]) ? trim($sar[9]) : "";
+	        } elseif ($id =='TT') {
+				//Claim automatic forward case.
+				$out[$ck]['clp'][$i]['crossover'] = 1;
+			}
+		} elseif ((strncmp('PER'.$de, $seg, 4) === 0 ) && ($segid == 'PER' && $out['loopid'] == '2100')) {
+			  $sar = explode($de, $seg);
+            $out['payer_insurance']  = trim($seg[2]);
+            $out['warnings'] .= 'Claim contact information: '.$seg[4];
+        } elseif (strncmp('PLB'.$de, $seg, 4) === 0 ) {
+			$sar = explode($de, $seg);
+			$p = (isset($out[$ck]['plb'])) ? count($out[$ck]['plb']) : 0;
+			$q = 0;
+			//
+			$out[$ck]['plb'][$p] = array(); 
+			$out[$ck]['plb']['adj'] = array();
+			$out[$ck]['plb'][$p]['provider'] = (isset($sar[1]) && $sar[1]) ? trim($sar[1]) : "";
+			$out[$ck]['plb'][$p]['fye'] = (isset($sar[2]) && $sar[2]) ? trim($sar[2]) : "";
+			// PLB02 is provider fiscal year end or CCYY1231
+			//
+			$plbar = array_slice($sar, 3);
+			foreach($plbar as $ky=>$plb) {
+				switch($ky % 2) {
+					// PLB04 06 08 ...
+					case 0:
+						$out[$ck]['clp'][$p]['adj'][$q]['amt'] = $plb;
+						break;
+					// PLB03 05 07 ...
+					case 1:
+						if (strpos($plb, $ds)) {				
+							$plb02 = explode($ds, $plb);
+							$out[$ck]['per'][$p]['adj'][$q]['code']  = $plb02[0];
+							$out[$ck]['per'][$p]['adj'][$q]['ref']  = $plb02[1];
+						} else {
+							$out[$ck]['per'][$p]['adj'][$q]['code']  = $plb;
+							$out[$ck]['per'][$p]['adj'][$q]['ref'] = '';
+						}
+						$q++;
+						break;
+				}
+			}
+			}
+			 $plbar .= (isset($sar[3]) && $sar[3]) ? trim($sar[3]) : '0';
+        // I am not sure that the assignment is corrent here, but based on the flow, I frame it. 
+
```

Comment:
Seems really odd to be adding this code in this script. What this meant to be changed?